integration/tests/common/mvp_test.go

```diff
@@ -124,13 +124,15 @@ func buildMVPNetConfig() testnet.NetworkConfig {
 		testnet.WithLogLevel(zerolog.InfoLevel),
 		// TODO replace these with actual values
 		testnet.WithAdditionalFlag("--access-address=null"),
+		testnet.WithAdditionalFlag("--insecure-access-api=true"),
```

Comment:
actually, can't we instead make the test actually work with `--insecure-access-api=false` to actually mimic how the production network will run?

Reply:
If it involved some investigation, we can issue it up but this test should be as close to the real world as possible.

Reply:
We can and initially that was the plan but there would be some investigation around orchestrating access node to come up before LN/SN nodes, also need to think about how this impacts total run time of the CI. We should issue it up or maybe have @gomisha look into it since he is already in that world.

Reply:
> investigation around orchestrating access node to come up before LN/SN nodes

I feel like any requirement we have for the AN to come up before the LN/SN would be true regardless of which API we are using. Does using the secure API change this?

> also need to think about how this impacts total run time of the CI

I also don't understand why using the secure API would impact the run time of the tests.

Maybe I'm missing something here.

Reply:
> I feel like any requirement we have for the AN to come up before the LN/SN would be true regardless of which API we are using. Does using the secure API change this? 

Currently I don't think any of the containers depends on or waits for any of the other containers to start up, we do this in localnet but not in this tests (must be verified). In any case the investigation is around how to get the access node ID and set that flag before we bring up LN/SN nodes. This will more than likely be different per test. 

> I also don't understand why using the secure API would impact the run time of the tests.
It depends on what we need to do as far as container orchestration.

Reply:
I am testing this out now